docs/blog/posts/constitutional-ai-or-validation.md

```diff
@@ -0,0 +1,221 @@
+---
+draft: False 
+date: 2023-10-17
+tags:
+    - pydantic
+    - validation
+    - guardrails 
+    - constitutional ai
+---
+
+# Constitutional AI is just Probabilistic Validation
+
+Earlier this year, there was a lot of discussion about Constitutional AI and its role in guiding models. There was also talk about the need for guard rails and special systems to ensure safe outputs. However, I found myself unimpressed. Why create a new term when we already have one that describes the concept? That term is "validation," and now we can do it probabilistically.
+
+Tools like Constitutional AI address shortcomings by using AI feedback to evaluate outputs. In this approach, an AI system provides a set of principles for making judgments about generated text. These principles guide the model's behavior. Essentially, it's a form of validation. Instead of manually checking against a set of rules, we now have a model that can perform the validation. For example, a content moderation system could use a model to automatically ban certain keywords. Previously, an error might have been thrown upon detection, but now we have a self-correcting system.
```

Comment:
Reading this again, I wonder if starting with this paragraph (and discarding the paragraph before) might be more punchy.